cypress/e2e/00-auth/t0-signup.cy.ts

```diff
@@ -5,21 +5,18 @@ describe('Sign up page test', () => {
     cy.visit('sign-up')
   })
 
-  it(
-    'should create a new user and redirect to the home after signup',
-    { defaultCommandTimeout: 5000 },
-    () => {
-      cy.visit('sign-up')
-
-      cy.get('input[name="organizationName"]').type('Company name')
-      cy.get('input[name="email"]').type(userEmail)
-      cy.get('input[name="password"]').type(userPassword)
-      cy.get('[data-test="submit-button"]').click()
-
-      cy.url().should('be.equal', Cypress.config().baseUrl + '/')
-      cy.get('[data-test="side-nav-name"]').contains('Company name')
-    }
-  )
+  it('should create a new user and redirect to the home after signup', () => {
+    cy.visit('sign-up')
+
+    cy.get('input[name="organizationName"]').type('Company name')
+    cy.get('input[name="email"]').type(userEmail)
+    cy.get('input[name="password"]').type(userPassword)
+    cy.get('[data-test="submit-button"]').click()
+
+    // TODO: Why is it redirecting to customer page and why is it forbidden?
```

Comment:
Should this be removed? I guess what you describe here is related to permissions, but it should not happen with a newly created account 🤔